test/behaviour/debug/bugs/type-resolver-bugs.feature

```diff
@@ -53,259 +53,288 @@ Feature: Graql Match Query
+#        $x isa house-number;
+#        $x 1.0;
+#      """
+#    Then answer size is: 1
+#    When get answers of graql query
+#      """
+#      match
+#        $x isa length;
+#        $x 2;
+#      """
+#    Then answer size is: 1
+#    When get answers of graql query
+#      """
+#      match
+#        $x isa attribute;
+#        $x >= 1;
+#      """
+#    Then answer size is: 2
+#    When get answers of graql query
+#      """
+#      match
+#        $x isa attribute;
+#        $x < 2.0;
+#      """
+#    Then answer size is: 1
+#
+#  @Ignore
+#  Scenario: when the answers of a value comparison include both a 'double' and a 'long', both answers are returned
+#    Given graql define
+#      """
+#      define
+#      length sub attribute, value double;
+#      """
+#    Given transaction commits
+#    Given the integrity is validated
+#    Given connection close all sessions
+#    Given connection open data session for database: grakn
+#    Given session opens transaction of type: write
+#    Given graql insert
+#      """
+#      insert
+#      $a 24 isa age;
+#      $b 19 isa age;
+#      $c 20.9 isa length;
+#      $d 19.9 isa length;
+#      """
+#    Given transaction commits
+#    Given the integrity is validated
+#    Given session opens transaction of type: read
+#    When get answers of graql query
+#      """
+#      match
+#        $x isa attribute;
+#        $x > 20;
+#      """
+#    Then uniquely identify answer concepts
+#      | x                 |
+#      | value:age:24      |
+#      | value:length:20.9 |
+#
+#    @Ignore
+#  Scenario: all relations and their types can be retrieved
+#    Given connection close all sessions
+#    Given connection open data session for database: grakn
+#    Given session opens transaction of type: write
+#    Given graql insert
+#      """
+#      insert
+#      $x isa person, has name "Bertie", has ref 0;
+#      $y isa person, has name "Angelina", has ref 1;
+#      $r (friend: $x, friend: $y) isa friendship, has ref 2;
+#      """
+#    Given transaction commits
+#    Given the integrity is validated
+#    Given session opens transaction of type: read
+#    Given get answers of graql query
+#      """
+#      match $r isa relation;
+#      """
+#    Given answer size is: 1
+#    Given get answers of graql query
+#      """
+#      match ($x, $y) isa relation;
+#      """
+#    # 2 permutations of the roleplayers
+#    Given answer size is: 2
+#    When get answers of graql query
+#      """
+#      match ($x, $y) isa $type;
+#      """
+#    # 2 permutations x 3 types {friendship,relation,thing}
+#    Then answer size is: 6
 
-  Scenario: when the answers of a value comparison include both a 'double' and a 'long', both answers are returned
-    Given graql define
-      """
-      define
-      length sub attribute, value double;
-      """
-    Given transaction commits
-    Given the integrity is validated
+  Scenario: 'iid' matches the instance with the specified internal iid
```

Comment:
I this scenario still broken?

Reply:
No, it's fixed. It's in matchTest so it is passes Grabl it's fine.